### DISCUSSION TITLE
Is there a light-theme out there?

### DISCUSSION THREAD

**Discussion by vadviktor** [starship/starship#2169] [Q&A]
I am using light (dark colours on white background) in my editors and terminals because it is much easier on my eyes. Some of the colours, like yellow are being hard to see on white. Does anyone know of a good light theme, please?

**Comment by mickimnet** [comment]
Working on it, please stay tuned for a couple of days.

**Reply by vadviktor** [reply]
Thank you @mickimnet :pray: 

FYI I am using Starship on Gnome Terminal and on Windows Terminal (both PowerShell and WSL2 Ubuntu), so I'll be able to give feedback on your work if it would help you perfect it. I'm using Python, Ruby, Node sections daily.

**Reply by mickimnet** [reply]
Btw, just a quick question, what is the value of ANSI yellow, that is configured in your terminal?

**Reply by vadviktor** [reply]
Hey, I'm using Tango Light colour set with light yellow at #FCE94F.

**Reply by mickimnet** [reply]
I was playing around with a light theme for my terminal as well, yesterday. And figured, that anything below #FFFF80 isn't really readable. Because the default _Starship_ uses the yellow terminal color.

PS: I'm still doing a light theme, in order to be a bit more independent from the terminal color set.

**Reply by vadviktor** [reply]
![Screenshot_2021-02-05_11-39-43](https://user-images.githubusercontent.com/238703/107029445-edfdd700-67a6-11eb-8b2d-b2d5a909bfd6.png)

@mickimnet just giving you a screenshot where I can show you how it looks to me currently. I use the Fish shell, tried to set its colours to a light theme to see it it would help, but it has no effect. I guess I'll have to dive into configuring that segment directly because it is only that duration segment that has this unreadable yellow colour.

**Reply by nickolay** [reply]
I had the same problem, and ended up changing iTerm's ANSI yellow to be a bit darker (thanks to https://github.com/starship/starship/issues/2388 ).
